src/Apps/CollectorProfile/Components/CollectorProfileHeader/CollectorProfileHeader.tsx

```diff
@@ -13,14 +16,27 @@ interface CollectorProfileHeaderProps {
 const CollectorProfileHeader: React.FC<CollectorProfileHeaderProps> = ({
   me,
 }) => {
+  const [localImage, setLocalImage] = useState<LocalImage>()
   const { name, createdAt, bio } = me
 
+  useEffect(() => {
+    getProfileLocalImage().then(image => {
+      if (image) {
+        setLocalImage(image)
+      }
+    })
+  }, [])
+
   return (
     <>
       <Spacer y={[2, 4]} />
 
       <Flex>
-        <CollectorProfileHeaderAvatarFragmentContainer me={me} />
+        {localImage ? (
+          <LocalImagePreview imageUrl={localImage.data} />
+        ) : (
+          <CollectorProfileHeaderAvatarFragmentContainer me={me} />
+        )}
```

Comment:
since we are getting the local profile image as part of the useEffect, this could result in the old image being shown first. How about we show a spinner here when it's loading.